common/dllserver/thorplugin.cpp

```diff
@@ -690,7 +690,7 @@ extern DLLSERVER_API bool decompressResource(size32_t len, const void *data, Str
 extern DLLSERVER_API void appendResource(MemoryBuffer & mb, size32_t len, const void *data, bool compress)
 {
     mb.append((byte)0x80).append(resourceHeaderVersion);
-    compressToBuffer(mb, len, data, compress ? COMPRESS_METHOD_LZW : COMPRESS_METHOD_NONE);
+    compressToBuffer(mb, len, data, compress ? COMPRESS_METHOD_LZ4HC : COMPRESS_METHOD_NONE);
```

Comment:
should this be a separate PR?

Reply:
yes.  This should also be retargetted back to 9.6.x, and the default should only be changed in master.